github/repos_pages.go

```diff
@@ -18,11 +18,11 @@ type Pages struct {
 	CNAME     *string      `json:"cname,omitempty"`
 	Custom404 *bool        `json:"custom_404,omitempty"`
 	HTMLURL   *string      `json:"html_url,omitempty"`
-	Source    *PageOptions `json:"source,omitempty"`
+	Source    *PagesSource `json:"source,omitempty"`
 }
 
-// PageOptions represents options to enable/disable pages
-type PageOptions struct {
+// PagesSource represents options to enable/disable pages
```

Comment:
I think this comment needs updating... how about something like:

```
// PagesSource represents a GitHub page's source.
```

Please make sure to end with a period for the autogenerated Godocs.